semantic/src/themes/universe/elements/button.overrides

```diff
@@ -86,3 +86,8 @@
 .ui.primary.button {
   background-color: @blueDark;
 }
+
+.ui.button:not(.icon) > .icon:not(.button):not(.dropdown)[class*="universe-chevron-right"] {
+  margin-left: 5px;
+  margin-right: -5px;
+}
```

Comment:
made this as specific as possible towards `universe-chevron-right` within buttons as to not regress other icon stylings elsewhere.

Reply:
Why is this only useful for the chevron?
Will other icons be styled correctly if put as a right icon in a button?

Reply:
for this one, I wanted to be specific towards the chevron just because I wasn't sure what the desired effect was for other icons. but I'll have a look at that now and see how it looks

Reply:
Yea, i totally get it... but if for whatever reason we can handle other icons as well (they should all be evenly spaced now) with this change, then it might be worth it to generalize the rule.

Should be enough to try on this branch if another random icon is styled appropriately.

Reply:
so I made it more general, and unfortunately I ran into this one on the `Tickets` page 👎 

![image](https://user-images.githubusercontent.com/29210883/58913430-2dc70e00-86ea-11e9-9ad1-07ba5803e619.png)

---

I can modify the rule from:
```
.ui.button:not(.icon) > .icon:not(.button):not(.dropdown)[class*="universe-chevron-right"] {
```
...to
```
.ui.button:not(.icon) > .icon:not(.button):not(.dropdown):not(.universe-plus) {
```

thoughts?